### ISSUE TITLE
2.53.16 is out

### ISSUE THREAD

**Issue by seemantk** [BioMike/gentoo-seamonkey-patches#1]
hello @BioMike !

Thank you for this repo. Wondering if you have any WIP patches for the new version of seamonkey (2.53.16)

TYSM

**Comment by BioMike** [comment]
Yes, I know 2.53.16 is out. There is just one big issue. The current ebuild fails when just switching the version number, due to some Python2.7/Python3 incompatibillity. Seamonkey still requires Python2.7, but support for it in portage seems to be broken somehow (building outside of portage works fine). As far as I know Seamonkey is the only package left that uses Python 2.7 in portage.

I'm already spending more than a year on porting Seamonkey to Python3, with the help of upstream, so I thought it would be better to see if I could make the full switch for Gentoo. This would 1) prevent the compatibillity issue I described above,  2) allow Gentoo to drop Python 2.7 completely and 3) Give me some indication on what's still missing for upstream to move to Python3 as well.

I started on this approach a few weeks ago, but progress is going slow. A python script that Seamonkey requires, but never was ported by Mozilla for Firefox, due to the fact that they didn't use that script any more has been taking most of that time. When I ported it to Python3 some issues surfaced and I'm spending past 2 weeks tracking those issues down. I'm still unsure how long this is going to take me to get this working. There is a good chance that 2.53.16 will be skipped.
